CONTRIBUTING.md

```diff
@@ -56,6 +63,15 @@ Before starting to work on your first RudderStack integration, it is highly reco
 * When developing a **source integration**, you’ll be transforming your events data received from the source to this specification.
 * When developing a **destination integration**, you’ll be parsing the event data according to this event spec and transforming it to your destination’s data spec.
 
+Follow through the next sections to learn about developing a new **source integration**
+
+* [Overview of integration development journey](#overview-of-integration-development-journey)
+* [1. Setup rudder-transformer and understand the code structure](#1-setup-rudder-transformer-and-understand-the-code-structure)
+* [2. Write code for a v0 source integration](#2-write-code-for-a-v0-source-integration)
```

Comment:
As v0 API got deprecated now, we can suggest to write in v1 spec

Reply:
Fixed it. Now we are directing contributors towards `src/v1/source` for source integration. But we still use example of `v0` source integration.

@krishna2020 What are the key differences between `v0` vs `v1` source integration a contributor must know? Context: Currently, I am using a v0 source integration example (Slack) in this guide, I want to understand what additional changes are required in the guide given that now we've updated steps to direct contributors towards `src/v1`?

Reply:
Another follow up question: Are we planning to migrate current v0 sources to v1?

Reply:
v1 will additionally have source config also as an input along with the event
We already highlighted this in the recent server transformer matrix docs

Reply:
Understood. The difference is in the `process` function arguments

* `v0` - `process(event)`
* `v1` - `process({ event: eventData, source: sourceConfigData }`

So the best solution is to simply use v1 source example to avoid any confusion. Which will require a major change in the content. How about we merge this PR for now, and then I raise another PR replacing v0 source example with v1 source example?